src/cargo/core/resolver/dep_cache.rs

```diff
@@ -167,6 +167,36 @@ impl<'a> RegistryQueryer<'a> {
     }
 }
 
+pub fn build_deps(
```

Comment:
Similar to below, could this document the return value and what each value of the tuple is?

Reply:
(also while you're at it if you can add general documentation for what this method is doing that'd be great!